### ISSUE TITLE
[NBug] Unable to parse object ID "".

### ISSUE THREAD

**Issue by coola** [gitextensions/gitextensions#6745]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.FormatException: Unable to parse object ID "".
   at GitUIPluginInterfaces.ObjectId.Parse(String s)
   at GitCommands.GitModule.GetCurrentCheckout()
   at GitUI.RevisionGridControl.ForceRefreshRevisions()
   at GitUI.CommandsDialogs.FormBrowse.RefreshRevisions()
   at GitUI.GitUIExtensions.<InvokeAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Environment
- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3416.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Fixed in 3.1.1

**Comment by gerhardol** [comment]
> Fixed in 3.1.1
The correction was this was the first merge after the release branch creation, so no. This will be included in 3.2

**Comment by RussKie** [comment]
Thank you for the correction

**Comment by gerhardol** [comment]
Duplicate of #6512 
Corrected in #6513, included in 3.2
